docs/about/roadmap.rst

```diff
@@ -0,0 +1,37 @@
+.. _about:
+
+Roadmap
+=======
+
+Overview
+--------
+
+As of autumn 2020 scipp is still in an "early preview" state.
+The overall plan for the coming releases is to:
+
+- Consolidate the existing generic functionality.
+- Add missing generic functionality to provide a more complex toolbox.
+- Weed out usability hurdles, to ensure the library's interface is easy to use.
+
+At this point we do not have a fixed release cycle.
+Instead we publish new releases when major feature additions or breaking changes have gone into scipp.
```

Comment:
Is it worth saying here that "unstable" bleeding edge builds are still available `conda install scipp -c scipp/label/dev`? Or maybe not as we state elsewhere.

Reply:
Indeed, it is in the installation page.